src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -575,8 +575,11 @@ def serving(self, inputs):
             is used, optionally only the last `decoder_input_ids` have to be input (see `past_key_values`).
 
             For translation and summarization training, `decoder_input_ids` should be provided. If no
-            `decoder_input_ids` is provided, the model will create this tensor by shifting the `input_ids` to the right
-            for denoising pre-training following the paper.
+            `decoder_input_ids` is provided, the model will automatically create this tensor by shifting:
+
+            - the `labels` to the right for text generation if `labels` is provided,
+            - the `input_ids` to the right for denoising pre-training following the paper if no `labels` is provided.
+
```

Comment:
```suggestion
            `decoder_input_ids` is provided, the model will automatically create this tensor by shifting:

            - the `labels` to the right for text generation if `labels` are provided,
            - the `input_ids` to the right for denoising pre-training following the paper if no `labels` are provided.
```suggestion
            `decoder_input_ids` is provided, the model will automatically create this tensor by shifting:

            - the `labels` to the right for text generation if `labels` are provided,
            - the `input_ids` to the right for denoising pre-training following the paper if no `labels` are provided.

```
```